webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -611,3 +711,12 @@ func (v validator) isNetworkRestrictedForReadinessProbe(ctx *context.WebhookRequ
 
 	return configMap.Data[isRestrictedNetworkKey] == "true", nil
 }
+
+func equalsOneOf[T comparable](a T, set ...T) bool {
```

Comment:
I don't see this being referenced anywhere in the diff?

Reply:
Yeah, I need to remove it. I did used it once, and then stopped.